v2/spiffebundle/set.go

```diff
@@ -0,0 +1,41 @@
+package spiffebundle
```

Comment:
Yes, that is a good thing. I'll probably wait to add one until we've settled on the API shape (since the package or its contents could change). Probably in a follow-up PR.